diff --git a/bcm2837-rpi-3-b.dts.orig b/bcm2837-rpi-3-b.dts
Upstream-Status: Pending
index 6127034..b645251 100644
--- a/arc/arm/boot/dts/broadcom/bcm2837-rpi-3-b.dts
+++ b/arch/arm/boot/dts/broadcom/bcm2837-rpi-3-b.dts
@@ -152,3 +152,15 @@
 &wifi_pwrseq {
 	reset-gpios = <&expgpio 1 GPIO_ACTIVE_LOW>;
 };
+
+&i2c1 {
+	pinctrl-name = "default";
+	pinctrl-0 = <&i2c1_gpio2>;
+	status = "okay";
+};
+
+&spi {
+	pinctrl-name = "default";
+	pinctrl-0 = <&spi1_gpio16>;
+	status = "okay";
+};
